include/highfive/H5Attribute.hpp

```diff
@@ -83,6 +83,14 @@ class Attribute : public Object {
     template <typename T>
     void write_raw(const T* buffer, const DataType& dtype = DataType());
 
+    static Attribute FromId(const hid_t& id){
+      Object obj = Object(id, ObjectType::Attribute);
+      return Attribute(obj);
```

Comment:
```suggestion
      return Attribute(Object(id, ObjectType::Attribute););
```
This would use the move constructor, and spare you from creating a new one. I wonder if this could be implemented generically in the object class.

Reply:
Do you mean we could remove the older constructor `explicit Object(hid_t);` and leave only `explicit Object(const hid_t&, const ObjectType&);` ? If so then I had this idea and I support it. If don't mind I could do that